tests/diff/collection_name_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package diff
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// TestCollectionName documents difference in reposnes:
+//   * dots
+//   * dashes
+//   * max length in FerretDB: 255, in MongoDB: 63.
+//   * FerretDB reserved prefix is _ferretdb_, MongoDB - system..
+func TestCollectionName(t *testing.T) {
+	t.Parallel()
+
+	ctx, db := setup(t)
+	db = db.Client().Database("admin")
+	dbName := db.Name()
+
+	t.Run("Dot", func(t *testing.T) {
```

Comment:
We really need only one of those tests, just like we have only one test for different error messages

Reply:
In the case of a regex, we don't even check all filtered cases. I tried to cover the most often cases. 
But OK..